src/order/filter/at_top_bot.lean

```diff
@@ -135,12 +135,32 @@ Ioi_mem_at_top a
 
 lemma eventually_ne_at_top [preorder α] [no_max_order α] (a : α) :
   ∀ᶠ x in at_top, x ≠ a :=
-(eventually_gt_at_top a).mono (λ x hx, hx.ne.symm)
+(eventually_gt_at_top a).mono $ λ x, ne_of_gt
+
+lemma eventually_gt_of_tendsto_at_top [preorder β] [no_max_order β] {f : α → β} {l : filter α}
```

Comment:
Perhaps `tendsto.eventually_gt_at_top` to allow dot notation?

Reply:
Feel free to adopt this PR. I usually use `h.eventually (eventually_??_at_top _)`. I've just noticed that we have lemmas that assume `[ordered_ring R]` for no reason.